src/foobar/admin.py

```diff
@@ -192,6 +192,25 @@ def changelist_view(self, request, extra_context=None):
         return response
 
 
+@admin.register(models.WalletLogEntry)
+class WalletLogEntryAdmin(admin.ModelAdmin):
+    list_display = ('date_created',
```

Comment:
We should reorder the columns so it is old balance, amount and then new balance. ![image](https://cloud.githubusercontent.com/assets/74944/23128080/caf68ede-f77d-11e6-8442-55bad8031b57.png)